pandas/core/window/rolling.py

```diff
@@ -412,7 +412,7 @@ def _get_roll_func(self, func_name: str) -> Callable:
             )
         return window_func
 
-    def _get_cython_func_type(self, func):
+    def _get_cython_func_type(self, func: str) -> Callable:
```

Comment:
If possible to add subtypes to `Callable` typically very helpful

Reply:
Unfortunately the arguments to the cython functions for rolling are still not entirely uniform so the argument list can't be typed consistently.